_pages/page-about.md

```diff
@@ -52,7 +52,7 @@ Then, students submit applications and candidates are interviewed. Qualified stu
 Starting in the summer, each Summer of Maps fellow manages two different nonprofit projects. The fellows are paid a stipend for their work, in addition to the opportunity to learn from mentors and Azavea staff members.
 
 
-### {{ site.fellowship.year }} Dates
+### important Dates
```

Comment:
"I" should be uppercase
<img width="496" alt="screen_shot_2018-08-29_at_4_03_11_pm" src="https://user-images.githubusercontent.com/5672295/44812467-3e6bb600-aba5-11e8-8372-c1d8811462e9.png">